ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration.tsx

```diff
@@ -68,10 +69,32 @@ export const AdvancedConfigurationForm = ({
       );
     }
   }, [clusterInfo]);
+
   const handleBlur = (value: string, fieldName: string, hasError: boolean) => {
     if (!hasError && !value.includes('/') && value !== '') {
       setValue(fieldName, `${value}/32`);
     }
+
+    // Check for duplicate entries
+    const existingEntries: Array<{ sourceRange?: string | undefined }> =
```

Comment:
@Tikdev00 let's try to do this at the schema level, using Zod.
The file is at `ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration-schema.ts`